VERSION.sh

```diff
@@ -10,7 +10,7 @@ vendor="389 Project"
 # PACKAGE_VERSION is constructed from these
 VERSION_MAJOR=2
 VERSION_MINOR=4
```

Comment:
Shouldn't this be 2.5.0, now that 2.4 is branched?

Reply:
It should :(